.ansible-lint

```diff
@@ -0,0 +1,2 @@
+skip_list:
+  - "204"
```

Comment:
Minor nit:  add a comment here explaining what `204` is so that we don't have to look it up.

Reply:
I go back and forth on this. It comes straight out of meta_skeleton: https://github.com/oasis-roles/meta_skeleton/blob/master/.ansible-lint

I've mostly landed on the side of "we don't really need a comment for this", but it might be worth opening a skeleton issue if you want to take a closer look at this.

Reply:
I created a PR for the meta_skeleton:

https://github.com/oasis-roles/meta_skeleton/pull/21